src/Microsoft.ML.StaticPipe/KMeansStatic.cs

```diff
@@ -16,9 +16,9 @@ namespace Microsoft.ML.StaticPipe
     public static class KMeansClusteringExtensions
     {
         /// <summary>
-        /// KMeans <see cref="ClusteringContext"/> extension method.
+        /// KMeans <see cref="ClusteringCatalog"/> extension method.
         /// </summary>
-        /// <param name="ctx">The clustering context trainer object.</param>
+        /// <param name="catalog">The clustering catalog trainer object.</param>
```

Comment:
>catalog [](start = 25, length = 7)

Great, thanks for fixing these as well, I'd kind of forgotten that we'd want to change the parameter names and descriptors here... and there sure are a lot of them. ;)